core/frontend/src/PerModelCategoryVisibility.ts

```diff
@@ -56,6 +59,13 @@ export namespace PerModelCategoryVisibility {
     addOverrides(fs: FeatureSymbology.Overrides, ovrs: Id64.Uint32Map<Id64.Uint32Set>): void;
   }
 
+  /** Props to describe per model category visibility. */
+  export interface Props {
+    modelId: string;
+    categoryIds: Id64Arg;
```

Comment:
Why `Id64Arg` vs `Iterable<Id64String>`?

Reply:
changed it to iterable<id64string>